docs/concepts/index.rst

```diff
@@ -30,7 +30,7 @@ Concepts and Terminology
         Execution time for a single quantum machine instruction (:term:`QMI`, or
         problem).
 
-        Learn more: :ref:`qpu_timing_breakdown_access`.
+        Learn more: :ref:`qpu_timing_breakdown_access`
```

Comment:
I'm still seeing periods for entries "adiabatic" and "Chimera", otherwise GLTM

Reply:
This is done now